lib/components/form/submissions.vue

```diff
@@ -170,22 +163,10 @@ export default {
         })
         .catch(() => {});
     },
-    download() {
-      this.downloading = true;
-      const path = `/forms/${this.form.xmlFormId}/submissions.csv.zip`;
-      this
-        .get(path, { responseType: 'blob' })
-        .finally(() => {
-          this.downloading = false;
-        })
-        .then(({ data }) => {
-          // Revoke the previous URL.
-          if (this.downloadHref !== '#')
-            window.URL.revokeObjectURL(this.downloadHref);
-          this.downloadHref = window.URL.createObjectURL(data);
-          this.$nextTick(() => this.$refs.downloadLink.click());
-        })
-        .catch(() => {});
+    // The `disabled` class on the <a> element does not prevent keyboard
+    // navigation.
+    clickDownloadButton(event) {
```

Comment:
you could consider making this a global jQuery thing instead;
`$('body').on('click', 'a.disabled', (event) => { event.preventDefault(); });`

Reply:
(but only if that semantic always holds, of course)

Reply:
Ah good idea, I'll do that!